### PR TITLE
reverted get started files -- header component

### LINKED ISSUE #8: Header Component
image

## Props:
1. Text
2. left or right (dictates if it will be on the left or right side of screen)

### Critical Issue:
This issue is important to be done quickly because it is used throughout the whole website

### PR DESCRIPTION
**Pull request by kayliezhao**
added header component & used for the how to get started
image

### CODE REVIEW

File: src/app/home/gets-started.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import GetStarted from "@/components/GetStarted";
+
+const Home = () => {
```
Comment (SafeDuck) [inline src/app/home/gets-started.js]: The folder and files shouldn't be created. Please remove. Next meeting will explain why with next.js routing system.

File: src/components/Header.tsx [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from "react";
 
-const Header = () => {
-  return <div>Header</div>;
+const Header = (props: { position: string; title: string }) => {
+  return (
+    <div
+      className={`w-1/8 flex flex-row border-b-4 border-t-4 border-white p-2 pl-[5%] pr-[5%] text-3xl text-white ${props.position === "left" ? "border-r-4" : "border-l-4"}`}
```
Comment (SafeDuck) [inline src/components/Header.tsx]: Are you sure `w-1/8` is a class in tailwind? 
Hover over it in vscode (tailwind intellisense)

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/components/Header.tsx (hunk 2 above)
Comment (SafeDuck) [inline src/components/Header.tsx]: Add font too.

**Comment by kayliezhao** [comment]
deleted the extra file, changed font
image

File: src/app/home/gets-started.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  return <GetStarted />;
```
Comment (SafeDuck) [inline src/app/home/gets-started.js]: delete this folder and files

File: src/components/home/GetStarted.tsx [hunk 4]
```diff
@@ -1,7 +1,21 @@
+import React from "react";
+import Header from "@/components/Header";
+
+const Example = (props: { txt1: string }) => {
+  return (
+    <div className="flex w-full flex-col gap-3 text-3xl">
+      {/* change to justify-end when positioning to right */}
+      <div className="flex w-full justify-start">
+        <Header position="left" title={props.txt1} />
```
Comment (SafeDuck) [inline src/components/home/GetStarted.tsx]: Revert this file back to it's original state

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by kayliezhao** [comment]
reverted get started

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**